apps/modernization-ui/src/apps/search/laboratory-report/result/ResultedTestDisplay.tsx

```diff
@@ -0,0 +1,61 @@
+import { LabTestSummary } from 'generated/graphql/schema';
+import { ReactNode } from 'react';
+import styles from './resulted-test.module.scss';
+type Props = {
+    test: LabTestSummary;
+};
+export const ResultedTestDisplay = ({ test }: Props) => {
+    const numericResult = (): ReactNode => {
```

Comment:
nit (non-blocking) : This could be `ReactNode | undefined` if you're really feeling crazy you could use `Maybe` from `utils` and make it `Maybe<ReactNode>`